pilot/Makefile

```diff
@@ -83,3 +83,8 @@ racetest:
 
 platform/kube/config:
 	@ln -s ~/.kube/config platform/kube/
+
+# Installs istioctl only - for debugging istioctl during dev work
+.PHONY: istioctl
+istioctl:
+	go install istio.io/istio/pilot/cmd/istioctl
```

Comment:
This needs to move to main Makefile.. we are getting rid of the individual make files